### PR TITLE
feat(rust): update outputs of common create commands

### PR DESCRIPTION
**Pull request by caoakleyii**
## Checks

- [ ] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [ ] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [ ] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/node/create.rs [hunk 1]
```diff
@@ -180,40 +180,74 @@ pub fn parse_launch_config(config_or_path: &str) -> Result<Config> {
...
+        let addr = SocketAddr::from_str(&cmd.tcp_listener_address)?;
 
-    print_query_status(&mut rpc, node_name, true, is_default).await?;
+        spawn_background_node(&opts, &cmd, addr).await?;
 
+        // let is_default = check_default(&opts, node_name);
+
+        let is_node_up = is_node_up(&mut rpc, true).await?;
+
+        // print_query_status(&mut rpc, node_name, true, is_default).await?;
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/node/create.rs]: Should we remove the commented lines?

**Review by adrianbenavides — DISMISSED** [review]
The outputs look really great 👏 ! I left a small comment

File: implementations/rust/ockam/ockam_command/src/node/create.rs (hunk 1 above)
Comment (caoakleyii) [reply implementations/rust/ockam/ockam_command/src/node/create.rs]: ah yep absolutely, thanks!

**Review by adrianbenavides — APPROVED** [review]
🚂

**Pull request merged**